<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreakpointDialog</class>
 <widget class="QDialog" name="BreakpointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Breakpoint</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Execution stopped at a breakpoint</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="continuePushButton">
        <property name="text">
         <string>Continue</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="singleStepPushButton">
        <property name="text">
         <string>Single Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="abortPushButton">
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>continuePushButton</sender>
   <signal>clicked()</signal>
   <receiver>BreakpointDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleStepPushButton</sender>
   <signal>clicked()</signal>
   <receiver>BreakpointDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>abortPushButton</sender>
   <signal>clicked()</signal>
   <receiver>BreakpointDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
